### DISCUSSION TITLE
Badge icon - where locatet / how to change

### DISCUSSION THREAD

**Discussion by iffinland** [GetPublii/Publii#1681] [General]
I would like to change the image of this icon. I searched the catalogs but couldn't find it. It turned out that it must be public somewhere because it is the same on all themes

![Screenshot 2024-09-11 08 21 05](https://github.com/user-attachments/assets/21087bd6-21f2-42c3-8a30-05a99744656b)

**Comment by GetPublii** [comment]
You can change it only via CSS, for example, by hiding the current one and adding a new icon to the cookie button through the background. The whole cookie banner can be easily restyled using CSS variables. Learn more about it in our [documentation](https://getpublii.com/docs/gdpr-cookie-banner-configuration.html#customizingtheappearanceofthecookiebanner).

**Comment by iffinland** [comment]
Thank you for answering! Unfortunately, I'm not a coder and I don't know css code. I would like to replace this icon with, for example, a picture like the one below in the post attachment.

![read-info-icon](https://github.com/user-attachments/assets/9f30c5f1-c5c8-4931-abea-6c0f13c8f034)
